NickvisionMoney.GNOME/Views/MainWindow.cs

```diff
@@ -456,8 +475,6 @@ private bool OnCloseAccountPage(Adw.TabView view, Adw.TabView.ClosePageSignalArg
             _viewStack.SetVisibleChildName("pageNoAccounts");
             _accountMenuButton.SetVisible(false);
             _flapToggleButton.SetVisible(false);
-            UpdateRecentAccountsOnStart();
-            _startPageRecentAccountsGroup.SetVisible(true);
```

Comment:
Shouldn't this be kept? The purpose of this wasn't to remove accounts from list, but reorder them if needed.

Reply:
I cleaned-up the code when rewriting the recent accounts, so this is no longer needed and it should still reorder like before.